src/helm/benchmark/run_specs/lite_run_specs.py

```diff
@@ -110,7 +110,26 @@ def get_commonsense_spec(dataset: str, method: str) -> RunSpec:
         metric_specs=get_exact_match_metric_specs(),
         groups=[dataset],
     )
+@run_spec_function("mmlu_pro")
+def get_mmlu_spec(subject: str, method: str = ADAPT_MULTIPLE_CHOICE_JOINT) -> RunSpec:
```

Comment:
Drop `method` - it will conflict with the chain of thought implementation later.